r/man/runDE.Rd

```diff
@@ -0,0 +1,32 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/differential_expression.R
+\name{runDE}
+\alias{runDE}
+\title{runDE function}
+\usage{
+runDE(req, data)
+}
+\value{
+Dataframe with columns:
+Gene
+avg_log2FC
+p_val_adj
+pct_1
+pct_2
+zscore          <-- Need to be returned until the UI is changed
+log2fc          <-- Need to be returned until the UI is changed
+pct             <-- Need to be returned until the UI is changed
+abszscore       <-- Need to be returned until the UI is changed
+qval            <-- Need to be returned until the UI is changed
+}
+\description{
+DE pipeline replicate
+}
+\details{
+Differential expression analysis can be requested. The request post include the cells ID that we need
+to compare in a cellSet named "base" and "background",
+For now, we are going to support Wilcoxon Rank Sum test, but we can add later other tests like "t", "MAST"
+or "DESeq2". We are going to return all the genes that have a absolute value avg_log2FC greater than 0.25 (this
+default value becomes from the Seurat default value of function FindMarkers)
+Out Gene, avg_log2FC,  p_val_adj,  pct_1, pct_2
+}
```

Comment:
What are these rd files? Are they automatically generated docs for our functions?

Reply:
Yes from `roxygen2`. They are used for `?function_name` documentation in build packages